bluebrain/repo-bluebrain/packages/model-neocortex/package.py

```diff
@@ -14,9 +14,11 @@ class ModelNeocortex(SimModel):
     """
 
     homepage = "https://bbpgitlab.epfl.ch/hpc/sim/models/neocortex"
-    git      = "ssh://git@bbpgitlab.epfl.ch/hpc/sim/models/neocortex.git"
+    git = "ssh://git@bbpgitlab.epfl.ch/hpc/sim/models/neocortex.git"
 
-    version('develop', branch='master', submodules=True, clean=False)
+    version('develop', branch='main', submodules=True, clean=False)
+    version('1.10', tag='1.10', submodules=True, clean=False)
+    version('1.9', tag='1.9', submodules=True, clean=False)
```

Comment:
what happened to 1.2 to 1.8?

Reply:
`model-neocortex` package is not used by anyone and hence missed updates for 1.2-1.8.
With neuron-9.0a we realised this and hence added latest releases.

We could add older releases but I think it's not necessary because 1) as mentioned, it's not used 2) I would like to just remove it with refactoring of neurodamus recipes.

Reply:
Why do we have this package if "it is not used by anyone"?

Reply:
> I would like to just remove it with refactoring of neurodamus recipes.

Great!

Reply:
> Why do we have this package if "it is not used by anyone"?

Fernando prototype this as way for MMB where multiple models (neocortex, hippocampus, thalamus) can be built  under single package (as multiple shared libraries). The idea was to use this for experiemtation and gathering feedback from users.

But now I think we might do this differently as we are also thinking to merge all neurodamus mod files into single repo.